modules/logging/filewriter.go

```diff
@@ -96,6 +98,15 @@ type FileWriter struct {
 	// it will be rotated.
 	RollSizeMB int `json:"roll_size_mb,omitempty"`
 
+	// Roll log file after some time
+	RollInterval time.Duration `json:"roll_interval,omitempty`
+
+	// Roll log file at fix minutes
+	RollAtMinutes []int `json:"roll_minutes,omitempty`
+
+	// Roll log file at fix time
+	RollAt []string `json:"roll_at,omitempty`
```

Comment:
I don't really understand what the inputs are here. What are the values supposed to be? The godoc comment here should be fleshed out.

Reply:
Answering my own question, it looks something like this in timberjack

```
RotateAt:         []string{"00:00", "12:00"}, // Also rotate at 00:00 and 12:00 each day
```

Reply:
Doc added for this config param in 6ff71267